pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -905,9 +866,9 @@ impl<T: AsRef<DeltaLayerInner>> ValueRef<T> {
     }
 }
 
-struct Adapter<T: AsRef<DeltaLayerInner>>(T);
+struct Adapter<'a>(&'a DeltaLayerInner);
 
-impl<T: AsRef<DeltaLayerInner>> BlockReader for Adapter<T> {
+impl BlockReader for Adapter<'_> {
     fn read_blk(&self, blknum: u32) -> Result<BlockLease, std::io::Error> {
         self.0.as_ref().file.read_blk(blknum)
```

Comment:
Well, this AsRef is worthless, there's an `impl AsRef<DeltaLayerInner> for DeltaLayerInner` as well which should go.

Reply:
This will be done in the next push.